### DISCUSSION TITLE
Authentik shows unused providers on dashboard

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [goauthentik/authentik#11825] [Q&A]
Heho
I only use LDAP as provider, but Authentik shows also SCIM, Google Workspace and Microsoft Entra as used providers.
![Bildschirmfoto vom 2024-10-26 15-59-24](https://github.com/user-attachments/assets/60f0e84a-672f-458e-affa-54242d79360a)
Is this a bug? I can't find them anywhere, but behind every provider is a 1.

Greetings 
Cy

**Comment by Raito00** [comment]
Hello!
The same problem.

![image](https://github.com/user-attachments/assets/c6b36338-5cce-4f0c-90a3-0e56a4bfd094)
